QA_Robot_Practice/services/LoginService.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
```

Comment:
Split services between their interfaces: UI, API, DB  and add a relevant postfix to service names as it is described in the Coding Standarts